travis-scripts/valgrind.sh

```diff
@@ -5,9 +5,23 @@ if [ -d /var/cfengine ]; then
     rm -rf /var/cfengine
 fi
 
+# Test assumes we start in core / masterfiles directory
```

Comment:
I'd write `or` instead of `/` because everywhere in this file `/` is a path separator so this looks like `masterfiles` must be subdir of `core`. But that's just a nitpick.

Reply:
https://github.com/cfengine/masterfiles/commit/16598847a1910d645b73ff81a65bbfcc8aa0ff0e
https://github.com/cfengine/core/pull/3853